rst/topics/failover.rst

```diff
@@ -158,6 +158,12 @@ algorithm is slightly different from that in case of eventual failover:
 * Every appointment (self-made or fetched) is immune for a while
   (controlled by the ``IMMUNITY_TIMEOUT`` option).
 
+You can also enable ``leader_autoreturn`` to return leadership to the
+first leader in ``failover_priority`` list after failover was triggered.
+It might be useful when you have active and passive data center.
+Time before failover will try to return leader is configured by
+``autoreturn_delay`` option in failover configuration.
```

Comment:
```suggestion
You can also enable ``leader_autoreturn`` to return leadership to the
first leader in ``failover_priority`` list after failover was triggered.
It might be useful when you have active and passive data centers.
The time before failover will try to return the leader is configured by
``autoreturn_delay`` option in a failover configuration.
```